prettier.config.js

```diff
@@ -0,0 +1,9 @@
+// https://prettier.io/docs/en/options.html
+module.exports = {
+  trailingComma: "es5",
+  bracketSpacing: true,
+  tabWidth: 2,
+  semi: false,
+  singleQuote: false,
+  arrowParens: "always",
+}
```

Comment:
This is the settings for things like semicolons, single-quotes etc for prettier which is the formatter